requirements.txt

```diff
@@ -1 +1,2 @@
 requests>=2.7.0
+pyhcl>=0.2.1,<0.3.0
```

Comment:
That is debatable, since the pyhcl library is used in the actual library if it is installed. It is an optional dependency in setup.py, but I think that for development purposes it makes sense to keep it in the regular requirements.txt